community_tasks/arabic_evals.py

```diff
@@ -196,7 +198,367 @@ def arabic_exams(line, task_name: str = None):
+    prompt_function="alghafa_prompt",
+    suite=["community"],
+    hf_repo="OALL/AlGhafa-Arabic-LLM-Benchmark-Translated",
+    hf_subset="arc_easy_ar",
+    hf_avail_splits=["test", "validation"],
+    evaluation_splits=["test"],
+    few_shots_split="validation",
+    few_shots_select="sequential",
+    metric=["loglikelihood_acc_norm"],
+    # metric=["loglikelihood_acc"],
+    trust_dataset=True,
+)
+
+
+# arc_challenge_okapi_ar
+arc_challenge_okapi_ar_task = LightevalTaskConfig(
+    name="arc_challenge_okapi_ar",
+    prompt_function="alghafa_prompt",
+    suite=["community"],
+    hf_repo="OALL/AlGhafa-Arabic-LLM-Benchmark-Translated",
+    hf_subset="arc_challenge_okapi_ar",
+    hf_avail_splits=["test", "validation"],
+    evaluation_splits=["test"],
+    few_shots_split="validation",
+    few_shots_select="sequential",
+    metric=["loglikelihood_acc_norm"],
+    # metric=["loglikelihood_acc"],
+    trust_dataset=True,
+)
+
+
+# mmlu_okapi_ar
+mmlu_okapi_ar_task = LightevalTaskConfig(
+    name="mmlu_okapi_ar",
+    prompt_function="alghafa_prompt",
+    suite=["community"],
+    hf_repo="OALL/AlGhafa-Arabic-LLM-Benchmark-Translated",
+    hf_subset="mmlu_okapi_ar",
+    hf_avail_splits=["test", "validation"],
+    evaluation_splits=["test"],
+    few_shots_split="validation",
+    few_shots_select="sequential",
+    metric=["loglikelihood_acc_norm"],
+    # metric=["loglikelihood_acc"],
+    trust_dataset=True,
+)
+
+
+# openbook_qa_ext_ar
+openbook_qa_ext_ar_task = LightevalTaskConfig(
+    name="openbook_qa_ext_ar",
+    prompt_function="alghafa_prompt",
+    suite=["community"],
+    hf_repo="OALL/AlGhafa-Arabic-LLM-Benchmark-Translated",
+    hf_subset="openbook_qa_ext_ar",
+    hf_avail_splits=["test", "validation"],
+    evaluation_splits=["test"],
+    few_shots_split="validation",
+    few_shots_select="sequential",
+    metric=["loglikelihood_acc_norm"],
+    # metric=["loglikelihood_acc"],
+    trust_dataset=True,
+)
+
+
+# boolq_ar
+boolq_ar_task = LightevalTaskConfig(
+    name="boolq_ar",
+    prompt_function="boolq_prompt_arabic",
+    suite=["community"],
+    hf_repo="OALL/AlGhafa-Arabic-LLM-Benchmark-Translated",
+    hf_subset="boolq_ar",
+    hf_avail_splits=["test", "validation"],
+    evaluation_splits=["test"],
+    few_shots_split="validation",
+    few_shots_select="sequential",
+    metric=["loglikelihood_acc_norm"],
+    # metric=["loglikelihood_acc"],
+    trust_dataset=True,
+)
+
+
+def boolq_prompt_arabic(line, task_name: str = None):
+    question = line["question"]
+    passage = line["passage"]
+    answer = "نعم" if line["answer"] else "لا"
+
+    query = 'بناءً على المقطع التالي:\n{}\n أجب عن هذا السؤال بـ "نعم" أو "لا":\n{}\nالإجابة:'.format(passage, question)
+
+    return Doc(
+        task_name=task_name,
+        query=query,
+        choices=["نعم", "لا"],
+        gold_index=0 if line["answer"] else 1,
+        instruction="",
+        target_for_fewshot_sorting=answer,
```

Comment:
I don't think you want to use `target_for_fewshot_sorting`, if your strategy for few shot is `sequential`